rclcpp/include/rclcpp/time.hpp

```diff
@@ -33,38 +35,67 @@ class Time
   static Time
   now()
   {
-    rcutils_time_point_value_t rcutils_now = 0;
-    rcutils_ret_t ret = RCUTILS_RET_ERROR;
+    // TODO(karsten1987): This impl throws explicitely on RCL_ROS_TIME
+    // we have to do this, because rcl_time_source_init return RET_OK
+    // however sets system time under the hood
```

Comment:
Can you add a sentence here about when we can remove this code, i.e. something like "this can be removed when the underlying implementation actually supports ROS time rather than implicitly defaulting to system time"?